src/Sylius/Bundle/ThemeBundle/Asset/Installer/OutputAwareAssetsInstaller.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Asset\Installer;
+
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class OutputAwareAssetsInstaller extends AssetsInstaller implements OutputAwareInterface
+{
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOutput(OutputInterface $output)
+    {
+        $this->output = $output;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function installAssets($targetDir, $symlinkMask)
+    {
+        if ($this->hasOutput()) {
+            if (AssetsInstallerInterface::HARD_COPY !== $symlinkMask) {
```

Comment:
Sure. You could extract method that returns only proper message, which is dependent on ``$symlinkMask`` value ;)